test/features/conditionalLogic.feature

```diff
@@ -58,4 +58,69 @@ Feature: Set of scenarios testing show & hide functionality and conditional logi
     When I do NOT meet the condition for showing the field in the tab
     Then the 'Text Field 3' field will NOT be visible on the 'Conditional Tab 1' tab
 
+  @OrAndNotEqualsLogic
+  Scenario: Case View field hide OR condition using case Text Field value
+    Given a case type containing conditionals exists for OR event
```

Comment:
OR event?
what does that mean?

Reply:
Looks like there are three case events introduced as part of these new tests 
createCaseNot (Create a case NOT)
createCaseOr (Create a case OR)
createCaseAnd (Create a case AND)